apps/frontend/src/pages/community/index.tsx

```diff
@@ -121,14 +121,6 @@ export default function CommunityPage() {
           communityIssuer,
           LeaderboardEntryType.TOTAL_TAP_COUNT
         );
-        weeklyDetails = await getUserLeaderboardDetails(
-          communityIssuer,
-          LeaderboardEntryType.WEEK_OCT_20_TAP_COUNT
-        );
-        weeklyEntries = await getTopLeaderboardEntries(
-          communityIssuer,
-          LeaderboardEntryType.WEEK_OCT_20_TAP_COUNT
-        );
```

Comment:
Rather than deleting old dashboards I wish they could be hidden somehow. I'm a sucker for digging into historical patterns / trends.

Reply:
yup! included in vivek's pr